css-device-adapt/Overview.bs

```diff
@@ -524,7 +524,9 @@ Computed value: auto, or a non-negative number or percentage as specified
 Specifies the largest allowed zoom factor. It is used as input to the
 <a href="#constraining-procedure">constraining procedure</a> to constrain
 non-''zoom/auto'' 'zoom' values, but also to limit the allowed zoom factor
-that can be set through user interaction. The UA should also use this
+that can be set through user interaction. Note that the UA may choose to ignore
```

Comment:
I originally intended it as an informative "this is what some UAs currently do anyway", but I guess at that point it makes sense to normatively allow it. I'll push a change to that effect.